packages/go/stbernard/command/generate/generate.go

```diff
@@ -81,6 +81,8 @@ func (s *command) Run() error {
 		return fmt.Errorf("generating code for workspace: %w", err)
 	} else if err := workspace.GenerateSchema(paths.Root, s.env); err != nil {
```

Comment:
Let's move GenerateSchema above WorkspaceGenerate to reduce issues with invalid code leading to not being able to generate schema, even though generating schema would be the only way to fix invalid code.